app/viewmodels/checkAnswers/euDetails/FixedEstablishmentTradingNameSummary.scala

```diff
@@ -27,17 +27,24 @@ import viewmodels.implicits._
 
 object FixedEstablishmentTradingNameSummary {
 
-  def row(answers: UserAnswers, index: Index)(implicit messages: Messages): Option[SummaryListRow] =
+  def row(answers: UserAnswers, index: Index, currentMode: Mode)(implicit messages: Messages): Option[SummaryListRow] = {
+
+    val changeLinkMode = currentMode match {
+      case NormalMode    => CheckLoopMode
+      case CheckMode     => CheckMode
```

Comment:
Could we ever be in CheckLoopMode at this point?

Reply:
No, and I'll add a case which throws an exception if that does occur